airflow/api_fastapi/core_api/routes/ui/graph.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Annotated
+
+from fastapi import Depends, Request, status
+from sqlalchemy.orm import Session
+
+from airflow.api_fastapi.core_api.datamodels.ui.graph import GraphDataResponse
+from airflow.api_fastapi.core_api.openapi.exceptions import create_openapi_http_exception_doc
+from airflow.utils.dag_edges import dag_edges
+from airflow.utils.task_group import task_group_to_dict
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Session
+from airflow.api_fastapi.common.db.common import get_session
+from airflow.api_fastapi.common.router import AirflowRouter
+
+graph_data_router = AirflowRouter(tags=["Graph"], prefix="/graph")
+
+
+@graph_data_router.get(
+    "/graph_data",
+    include_in_schema=False,
+    responses=create_openapi_http_exception_doc([status.HTTP_400_BAD_REQUEST]),
+)
+def graph_data(
+    session: Annotated[Session, Depends(get_session)],
+    dag_id: str,
+    request: Request,
+    root: str | None = None,
+    include_upstream: bool = False,
+    include_downstream: bool = False,
```

Comment:
This has been changed from `filter_upstream` and `filter_downstream` to `include_upstream` and `include_downstream`.

I find the original names confusing as this is actually doing the opposite. If `filter_downstream` is True, then we include the downstream dependencies...

Not sure if i'm misunderstanding something @bbovenzi